text/stacks/0000-node-on-tiny.md

```diff
@@ -50,4 +57,4 @@ None
 ## Unresolved Questions and Bikeshedding
 
 - Is it worth to influence other users of the stack?
-- Currently the `npm start` buildpack requires a `bash`, so any Node.js application using a start script would fail.
+- Currently the `npm start` buildpack requires a `bash`, so any Node.js application using a start script would fail, but this might be changed in `npm start`
```

Comment:
```suggestion
- Currently the `npm start` buildpack requires a `bash`, so any Node.js application using a start script would fail, but this can be changed in `npm start`
```

Maybe this makes more clear that we actively look into solving thod.